### PR TITLE
May be case not authorized to use Back Camera.

### PR DESCRIPTION
**Pull request by lgw51**

### CODE REVIEW

**Comment by lgw51** [comment]
Update ViewController.swift

File: Example/QRCodeReader.swift/ViewController.swift [hunk 1]
```diff
@@ -34,24 +34,31 @@ class ViewController: UIViewController, QRCodeReaderViewControllerDelegate {
...
+        present(reader, animated: true, completion: nil)
+      }
+      else {
+        let alert = UIAlertController(title: "Error", message: "Reader not supported by the current device", preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "OK", style: .cancel, handler: nil))
 
-      present(alert, animated: true, completion: nil)
+        present(alert, animated: true, completion: nil)
+      }
+    } catch let error as NSError {
```
Comment (yannickl) [inline Example/QRCodeReader.swift/ViewController.swift]: Why do you catch the error here if you don't use it?

I prefer the previous version.

**Comment by yannickl** [comment]
Thank you for your contribution! I'm waiting for your answer before to merge.

**Comment by lgw51** [comment]
Sorry, i just make a simple note, not finished it.

**Comment by lgw51** [comment]
i have make a new commit request

**Comment by yannickl** [comment]
Great, thank you. I'll update minor things tomorrow. 👍

**Pull request merged**